repo/packages/C/cassandra/1/command.json

```diff
@@ -0,0 +1,6 @@
+{
+  "pip": [
+    "dcos<1.0",
+    "git+https://github.com/mesosphere/dcos-cassandra.git#dcos-cassandra=0.1.0"
+  ]
```

Comment:
the automated build -> whl isn't set up yet, so lets use it as is for now and switch when we updated the dcos-cassandra repo?